--- a/Makefile.in
+++ b/Makefile.in
@@ -5852,8 +5852,8 @@
 	$(V_BUNDLE) cd $(srcdir) && NODE_ENV='$(NODE_ENV)' ./build.js
 install-bundles:
 	cd $(srcdir)/dist; find */* -type f ! -name '*.LEGAL.txt' -a ! -path 'playground/*' -exec install -D -m 644 '{}' '$(abspath $(DESTDIR)$(datadir))/cockpit/{}' \;
-	find $(srcdir)/dist -name '*.LEGAL.txt' -exec install --target-directory '$(abspath $(DESTDIR)$(docdir))/legal' '{}' -D -m 644 '{}'  \;
+	find $(srcdir)/dist -name '*.LEGAL.txt' -exec install --target-directory '$(abspath $(DESTDIR)$(docdir))/legal' -D -m 644 '{}' \;
 uninstall-bundles:
 	rm -rf $(DESTDIR)$(datadir)/cockpit
 
 # Extract translate attribute, Glade style, angular-gettext HTML translations
